### PR TITLE
Modified the query file with more sql queries

### PR DESCRIPTION
**Pull request by medsonmoombe**
## In this pull request the following were implemented:

- Added a column species of type string to  animals table. Modified the schema.sql file.
- Inserted the following data and added the insertions in data.sql file.:
    - Animal: His name is Charmander. He was born on Feb 8th, 2020, and currently weighs -11kg. He is not neutered and he has never tried to escape.
    - Animal: Her name is Plantmon. She was born on Nov 15th, 2021, and currently weighs -5.7kg. She is neutered and she has tried to escape 2 times.
    - Animal: His name is Squirtle. He was born on Apr 2nd, 1993, and currently weighs -12.13kg. He was not neutered and he has tried to escape 3 times.
    - Animal: His name is Angemon. He was born on Jun 12th, 2005, and currently weighs -45kg. He is neutered and he has tried to escape once.
    - Animal: His name is Boarmon. He was born on Jun 7th, 2005, and currently weighs 20.4kg. He is neutered and he has tried to escape 7 times.
    - Animal: Her name is Blossom. She was born on Oct 13th, 1998, and currently weighs 17kg. She is neutered and she has tried to escape 3 times.
   -  Animal: His name is Ditto. He was born on May 14th, 2022, and currently weighs 22kg. He is neutered and he has tried to escape 4 times.
- Inside a transaction updated the animals table by setting the species column to unspecified. Verified that change was made. Then rolled back the change and verified that the species columns went back to the state before the transaction.

Rollback

- Inside a transaction:
    - Updated the animals table by setting the species column to digimon for all animals that have a name ending in mon.
    - Updated the animals table by setting the species column to pokemon for all animals that don't have species already set.
    - Committed the transaction
    - Verified that change was made and per
DELETan
sists after commit.
    
UPDATEspecies
- Deleted all records in the animals table, then roll back the transaction.

DELETROLLBACK

- Inside a transaction:
    - Deleted all animals born after Jan 1st, 2022.
    - Created a savepoint for the transaction.
deleBORN

    - Updated all animals' weight to be their weight multiplied by -1.
    - Roll backed to the savepoint
UPDATE-1WEIGHT

    - Updated all animals' weights that are negative to be their weight multiplied by -1.
    - Committed transaction
weight 0

- Wrote queries to answer the following questions:
    - How many animals are there?
total_animals

    - How many animals have never tried to escape?
zeroattempts

    - What is the average weight of animals?
avgweight

    - Who escapes the most, neutered or not neutered animals?
naturedEscapeA

    - What is the minimum and maximum weight of each type of animal?
SPECIESminMAX

    - What is the average number of escape attempts per animal type of those born between 1990 and 2000?

AVGESCAPT

- Took the screenshots for each query

### CODE REVIEW

File: queries.sql [hunk 1]
```diff
@@ -8,3 +8,41 @@ SELECT name, escape_attempts from animals WHERE weight_kg > 10.5;
...
+ROLLBACK TO FIRST_POINT;
+UPDATE animals SET weight_kg = weight_kg * (-1) WHERE weight_kg < 0;
+COMMIT;
+SELECT * FROM animals;
+
+-- Write queries to answer the following questions:
+SELECT COUNT(*) as total_animals FROM animals;
+SELECT COUNT(*) as zero_escape_attempts FROM animals WHERE escape_attempts = '0';
+SELECT AVG(weight_kg) as average_animal_weight FROM animals;
+SELECT neutered FROM animals WHERE escape_attempts = (SELECT MAX(escape_attempts) FROM animals);
```
Comment (gemmen29) [inline queries.sql]: - [ ] Kindly fix the query of `Who escapes the most, neutered or not neutered animals` as now you only select one animal was the most escape attempts.

**Review by gemmen29 — CHANGES_REQUESTED** [review]
Hi @medsonmoombe, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight 💯 
- Descriptive pull request ✔️ 
- Descriptive commit messages ✔️ 

## Required Changes ♻️
_Check the comments under the review._



## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by medsonmoombe** [comment]
## The following were implemented as requested by the reviewer:
-  fixed the query of Who escapes the most, neutered or not neutered animals .
Neutred11

**Review by ShoiraTa — APPROVED** [review]
## **STATUS: APPROVED  🟢 ** 
------
### Hi @medsonmoombe , 

**Great work on making the changes requested by a previous reviewer 👏🏻** 

Your project is complete! . There is nothing else to say other than... it's time to merge it 🍾🚢 :shipit: . Congratulations!  🎉💯🌟 

5SW

**Cheers and Happy coding! :clap: :clap: :clap:**

Feel free to leave any questions or comments in the PR thread by mentioning my Github username if something is not 100% clear.
***
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have  limited number of  reviews per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**